examples/bin/cluster-syntheticcontrol.sh

```diff
@@ -75,11 +75,21 @@ if [ "$HADOOP_HOME" != "" ] && [ "$MAHOUT_LOCAL" == "" ]; then
     echo "DFS is healthy... "
     echo "Uploading Synthetic control data to HDFS"
     $DFSRM ${WORK_DIR}/testdata
-    $DFS -mkdir ${WORK_DIR}/testdata
+    $DFS -mkdir -p ${WORK_DIR}/testdata
```

Comment:
Hmm.. I suppose not :).  I guess that's something to look at. I'm not sure when those crept in.  (I may have done it myself).  Anyway I'm not sure that Hadoop 1 incompatibility is a dealbreaker going forward- at least not in the example scripts, This is something that we need to discuss on dev@.  Thanks for pointing this out.  I will bring it up.